packages/legend-query-builder/src/components/__tests__/QueryBuilderFetchStructure.test.tsx

```diff
@@ -1329,112 +1320,6 @@ test(
   },
 );
 
-test(
```

Comment:
Moved this test and the below one to `QueryBuilderFilterPanel.test.tsx` since they are tests with the filter panel and not with the fetch structure.